### ISSUE TITLE
Aegir Leaves Drupal

### ISSUE THREAD

**Issue by omega8cc** [omega8cc/boa#1781]
In non-tech speak, people don’t seem to appreciate the speed and direction of Drupal developers voyage anymore, so we have decided to fix that — https://gist.github.com/omega8cc/2f3d68f45a003fbfd4690f2f5ee1cc93

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by aDarkling** [comment]
Wow - its refreshing to not have Dries et al. blow smoke up my butt about this!

I don't make any money from these sites right now, so won't need full functionality, but I am hoping to purchase a Pro license in recognition of the hard work you've done all these years.  

As long as its something I can afford.  Any updates on your pricing yet?  BTW, I deal in USD.

**Comment by omega8cc** [comment]
Thank you for your words @aDarkling it’s really appreciated! We haven’t published the Pro pricing variants yet because for now it’s almost identical to Lite, but it should be available later this week.

**Comment by omega8cc** [comment]
We have opened feature request in the Backdrop issue tracker: https://github.com/backdrop/backdrop-issues/issues/6453#issue-2244191235

**Comment by landNull** [comment]
> We haven’t published the Pro pricing variants yet because for now it’s almost identical to Lite, but it should be available later this week.

Just to clarify a few things,

Our family owned marketing agency migrated from aegir to boa this summer and as per the instructions installed lts. 

Is lts and lite the same thing?

Is this the thread where updates about Backport support will be shared?

**Comment by omega8cc** [comment]
@landNull Yes, LTS is the new name for previous LITE, STABLE OR HEAD. It's explained in [DUALLICENSE](https://github.com/omega8cc/boa/blob/5.x-dev/DUALLICENSE.md)

We will announce major developments in the [CHANGELOG](https://github.com/omega8cc/boa/blob/5.x-dev/CHANGELOG.txt)

Also, we have a dedicated document to track our progress: [ROADMAP](https://github.com/omega8cc/boa/blob/5.x-dev/ROADMAP.md)

**Comment by landNull** [comment]
@omega8cc Thank you for your patience, and helping us all find our way around the community here. I really appreciate you all taking the time to point us in the right direction.  I know you all have a lot going on which makes it all that more meaningful when you take time out to reply so quickly, and post links.  :)
